packages/twenty-front/src/modules/settings/data-model/constants/DatabaseIdentifierMaximumLength.ts

```diff
@@ -1 +1 @@
-export const DATABASE_IDENTIFIER_MAXIMUM_LENGTH = 63;
+export const DATABASE_IDENTIFIER_MAXIMUM_LENGTH = 62;
```

Comment:
What changed? It seems like we need to have objects no longer than 61 (62 - 1) characters long, I can't figure out why.
Getting errors like 

 QueryFailedError: type "_aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaao" already exists